aries-site/src/examples/components/tag/TagAnatomy.js

```diff
@@ -0,0 +1,130 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Grid, Diagram, Stack, Text, ThemeContext } from 'grommet';
+import { FormClose } from 'grommet-icons';
+import { Annotation } from '../namevaluelist/NameValueListAnatomy';
+
+const color = 'border';
+const anchor = 'vertical';
+const thickness = 'hair';
+const type = 'direct';
+
+const connections = [
+  {
+    anchor,
+    type,
+    color,
+    thickness,
+    fromTarget: 'border-annotation',
+    toTarget: 'border',
+  },
+  {
+    anchor: 'horizontal',
+    type,
+    color,
+    thickness,
+    fromTarget: 'name-annotation',
+    toTarget: 'tagName-2',
+  },
+  {
+    anchor,
+    type,
+    color,
+    thickness,
+    fromTarget: 'value-annotation',
+    toTarget: 'value',
+  },
+  {
+    anchor: 'horizontal',
+    type,
+    color,
+    thickness,
+    fromTarget: 'close-annotation',
+    toTarget: 'close',
+  },
+];
+
+export const AnatomyGrid = ({ ...rest }) => (
+  <Grid
+    columns={['xsmall', 'small', 'xsmall']}
+    gap={{ column: 'small', row: 'medium' }}
+    justify="center"
+    {...rest}
+  />
+);
+
+export const AnatomyBox = ({ id, nameId, valueId, closeId }) => {
+  const theme = useContext(ThemeContext);
+
+  return (
+    <Box
+      direction="row"
+      gap="xxsmall"
+      id={id}
+      border={theme.tag.border}
+      pad={theme.tag.pad}
+      round={theme.tag.round}
+    >
+      <Box id={nameId}>Location</Box>
+      {valueId && closeId && (
+        <>
+          <Box>:</Box>
+          <Box id={valueId}>
+            <Text {...theme.tag.value}>NY_USA</Text>
+          </Box>
+          <Box id={closeId}>
+            <FormClose />
+          </Box>
+        </>
+      )}
+    </Box>
+  );
+};
+
+export const TagAnatomy = () => {
+  const theme = useContext(ThemeContext);
+
+  return (
+    <Stack>
+      <AnatomyGrid>
+        {/* Empty Box occupies first cell of grid. Alternatively 
+        could use gridAreas, but felt heavy for this need. */}
+        <Box />
+        <Annotation id="border-annotation" target="1" />
+        <Box />
+        <Box alignSelf="center">
+          <Annotation id="name-annotation" target="2" />
+        </Box>
```

Comment:
Are these Boxes necessary? Can `Annotation` get a `...rest` and we can pass in `<Annotation alignSelf="center" ... />`?

Reply:
Good cleanup, I'll try.

Reply:
What was the outcome of this? I'm not seeing it implemented. Perhaps I am on the wrong commit? Or this commit didn't get pushed?

Reply:
You might not be looking at the latest, I incorporated this.